packages/shared/src/components/comments/CommentContainer.tsx

```diff
@@ -145,8 +149,14 @@ export default function CommentContainer({
                 @{comment.author.username}
               </TruncateText>
             </ProfileLink>
-            <div className="mx-2 h-0.5 w-0.5 bg-text-quaternary" />
+            <Separator />
             <CommentPublishDate comment={comment} />
+            {topReader && (
+              <>
+                <Separator />
+                <TopReaderIn topReader={topReader} tooltip />
```

Comment:
We might have to truncate this whole row? Probably noticeable on mobile devices.

Reply:
Can you add these comments to #3739?

Reply:
I just got back to my GH emails 🙈 

I will check if this hasn't been resolved.